tools/scxtop/src/app.rs

```diff
@@ -302,11 +300,19 @@ impl<'a> App<'a> {
     }
 
     /// Stop all active perf events.
-    fn stop_perf_events(&mut self) {
+    fn stop_perf_events(&mut self) -> Result<()> {
         for cpu_data in self.cpu_data.values_mut() {
             cpu_data.data.clear();
         }
         self.active_perf_events.clear();
+
+        self.available_events = PerfEvent::default_events();
```

Comment:
Maybe this could be moved into a helper `reset_perf_events` as stopping seems like it should be logically separated from resetting.